tests/commands/test_pipelines_commands.py

```diff
@@ -233,3 +233,29 @@ def test_get_info_api_exception_different_401(capture_logs, unstub):
     assert SUPPORT_EMAIL_TEXT in capture_logs.text
 
     unstub()
+
+
+def test_non_json_api_exception_body(capture_logs, unstub):
+    runner = CliRunner()
+    pipeline_name = "whatever"
+
+    when(pipelines_commands.pipelines_logic).get_pipeline_info(
+        pipeline_name, None
+    ).thenRaise(
+        ApiException(
+            status=401,
+            reason=None,
+            body='403 Forbidden: blah blahbalh',
```

Comment:
Is the `blah blahbalh` just a place holder or a joke I'm not getting?

Reply:
yea the text in here doesnt actually matter, just need a body that isnt a json